Source/Core/DolphinQt/Settings/GameCubePane.cpp

```diff
@@ -186,12 +189,25 @@ void GameCubePane::OnConfigPressed(int slot)
   {
     bool ok;
     const auto new_mac = QInputDialog::getText(
-        this, tr("Broadband Adapter MAC address"), tr("Enter new Broadband Adapter MAC address:"),
-        QLineEdit::Normal, QString::fromStdString(SConfig::GetInstance().m_bba_mac), &ok);
+        this, tr("Broadband Adapter (TAP) MAC address"),
```

Comment:
Probably a i18n comment explaining what the terms "TAP" and "MAC" would be wise.